intermediate/remote_data_access.ipynb

```diff
@@ -0,0 +1,331 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    df = pd.read_csv("https://storage.googleapis.com/cmip6/cmip6-zarr-consolidated-stores.csv")
</pre><p>This is potentially an outdated catalog file. I need to "clean house" at some point, but in the meantime this is the "officially" maintained csv: "https://cmip6.storage.googleapis.com/pangeo-cmip6.csv"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xarray-contrib/xarray-tutorial/pull/132/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Good to know, I've updated the CSV link in e96181b89f369e1358ee421de0529cfee5429a48